### PR TITLE
[DR-3013] Add TPS env variables

### PR DESCRIPTION
**Pull request by samanehsan**
https://broadworkbench.atlassian.net/browse/DR-3032

Add the `TPS_BASE_PATH` and `TPS_ENABLED` for dev, personal dev deployments, integration and perf. These environment variables were added in https://github.com/DataBiosphere/jade-data-repo/pull/1454 for integration with the Terra Policy Service. Currently, there is only a dev environment.

There is a follow-on ticket to update alpha, staging and prod once those TPS deployments exist: https://broadworkbench.atlassian.net/browse/DR-3035

### CODE REVIEW

**Review by okotsopoulos — APPROVED** [review]
Looks good!  Will there be a follow-on PR and/or ticket for setting these variables in production (not sure if TPS exists in prod yet)?

File: perf/datarepo/datarepo-api.yaml [hunk 1]
```diff
@@ -28,6 +28,8 @@ env:
   OIDC_ADDCLIENTIDTOSCOPE: true
   OIDC_EXTRAAUTHPARAMS: prompt=login
   SAM_OPERATIONTIMEOUTSECONDS: 120
+  TPS_ENABLED: true
+  TPS_BASE_PATH: https://tps.dsde-dev.broadinstitute.org
```
Comment (nmalfroy) [inline perf/datarepo/datarepo-api.yaml:32]: Just want to make sure: will there be an integration or perf deployment for TPS?

**Review by nmalfroy — COMMENTED** [review]
I have a quick question about deployments.  And just calling out: we're doing a separate ticket for the alpha/stage/prod envs right?

File: perf/datarepo/datarepo-api.yaml (hunk 1 above)
Comment (samanehsan) [reply perf/datarepo/datarepo-api.yaml:32]: I'll double check with the identiteam!

**Comment by samanehsan** [comment]
Yes! I decided to create a separate ticket to update alpha, staging and prod: https://broadworkbench.atlassian.net/browse/DR-3035. I updated the original ticket, but forgot to mention that in the PR description!

**Pull request merged**